modules/ROOT/pages/con_pipeline-cluster-resource.adoc

```diff
@@ -0,0 +1,48 @@
+[]
+= Cluster Resource
+
+
+A Cluster Resource represents a Kubernetes cluster other than the current cluster Pipelines is running on. A common use case for this resource is to deploy your application or function on different clusters.
+
+The resource will use the provided parameters to create a
+[kubeconfig](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/)
+file that can be used by other steps in the pipeline task to access the target
+cluster. The kubeconfig will be placed in `/workspace/<your-cluster-name>/kubeconfig` on your Task container
+
+The Cluster resource has the following parameters:
+
+* Required:
+*  `name` (required): The name to be given to the target cluster, will be used
+    in the kubeconfig and also as part of the path to the kubeconfig file
```

Comment:
Since these are two different points, let's break this into two sentences. "This name is used in the `kuberconfig` file and is also part of the path to the file.
@joaedwar: Is the name of the target cluster part of the kuberconfig file name? If so, we have to specify that.